src/libPMacc/examples/gameOfLife2D/include/GatherSlice.hpp

```diff
@@ -20,18 +21,15 @@
  */
 
 
-#ifndef GATHERSLICE_HPP
-#define	GATHERSLICE_HPP
+#pragma once
 
-#include "types.h"
 
 
 #include <mpi.h>
 #include "mappings/simulation/GridController.hpp"
-
-//c includes
 #include "memory/boxes/PitchedBox.hpp"
 #include "dimensions/DataSpace.hpp"
+#include "types.h"                                  // DIM*
```

Comment:
I never thought about changing the order but it sounds good for me.
@ComputationalRadiationPhysics/picongpu-maintainers We should add this rules for the include order to our [coding guideline](https://github.com/ComputationalRadiationPhysics/picongpu/wiki/Coding-Guide-Lines).